plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -4893,7 +4894,7 @@ protected static int getSplitCount(ConnectorSplitSource splitSource)
 
     protected static ConnectorSplitSource getSplits(ConnectorSplitManager splitManager, Transaction transaction, ConnectorSession session, ConnectorTableHandle tableHandle)
     {
-        return splitManager.getSplits(transaction.getTransactionHandle(), session, tableHandle, UNGROUPED_SCHEDULING, DynamicFilter.EMPTY);
+        return splitManager.getSplits(transaction.getTransactionHandle(), session, tableHandle, UNGROUPED_SCHEDULING, DynamicFilter.EMPTY, Constraint.alwaysTrue(), RetryMode.NONE);
```

Comment:
Yeah I guess. Though typically we extend the method signatures adding new parameters at the end. I am no super convinced I should change that. But I can if you prefer.

Reply:
>  Though typically we extend the method signatures adding new parameters at the end

That's nice when there is no compelling reason to do otherwise, but in fact there is nothing forcing us to so.
It's just easier to read that way. But there can be other readability pros/cons too (like i think is here)

Reply:
Fine - will move.